examples/ex29p.cpp

```diff
@@ -93,13 +94,7 @@ int main(int argc, char *argv[])
    args.AddOption(&visualization, "-vis", "--visualization", "-no-vis",
                   "--no-visualization",
                   "Enable or disable GLVis visualization.");
-   args.Parse();
-   if (!args.Good())
-   {
-      args.PrintUsage(cout);
-      return 1;
-   }
-   args.PrintOptions(cout);
+   args.ParseCheck();
```

Comment:
Same here.

Reply:
I think this "Example 0" version is cleaner, particularly in parallel.

I don't feel so strongly about it though... do you insist we revert?

Reply:
Depends on what others think? @mlstowell, do you have a preference?

Reply:
I don't have a strong opinion. We can keep the shorter implementation.

Reply:
Looks to me there is a function to do exactly what was done repetitively before. I like that it is being used and is much shorter, especially for an arg check: I don't think that is what people came looking for in an example.

Reply:
Okay, let's keep the short version.